src/features/groups/components/ContainerCardList/ContainerCard/RenameContainerForm.tsx

```diff
@@ -80,6 +85,9 @@ export const RenameContainerForm = ({
    * @param event - The keyboard event
    */
   const handleKeyDown = (event: KeyboardEvent<HTMLInputElement>) => {
+    if (isLoading) {
+      return; // Do nothing if isLoading is true
+    }
```

Comment:
This part is unnecessary because [ escape ] key down does not trigger a submit.
Multiple submissions by [ enter ] key down can be avoided within the processsSubmit, which you have already modified.